pkg/operator/azurepathfixcontroller.go

```diff
@@ -90,31 +66,8 @@ func NewAzurePathFixController(
 	}
 	c.cachesToSync = append(c.cachesToSync, imageRegistryConfigInformer.Informer().HasSynced)
 
-	if _, err := infrastructureInformer.Informer().AddEventHandler(c.eventHandler()); err != nil {
```

Comment:
Just to confirm, we have removed this because we check only for the infra type now, which cannot change, right?

Reply:
Yes, we only need the infrastructure object to check for the platform type which I don't expect to change: https://github.com/openshift/cluster-image-registry-operator/blob/92d64a771380eab0069fca9e4db82c784874bb4c/pkg/operator/azurepathfixcontroller.go?plain=1#L168-L174

Looking at the api implementation for PlatformStatus.Type I can't see any validation that the field cannot be updated, but I cannot think how a cluster's platform can change at runtime in a supported way. Source: https://github.com/openshift/api/blob/caf97963ed302881f01e67b791ae4efc6bcf723c/config/v1/types_infrastructure.go#L391-L402

Let me know if I'm missing something.

Reply:
I've updated the code, adding a comment explaining why we don't need an event handler for infrastructure object changes.

Reply:
Ok, cool.